### ISSUE TITLE
Split out language files into a separate module

### ISSUE THREAD

**Issue by rramphal** [indic-transliteration/sanscript.js#16]
@vvasuki suggested that we split out the language files into another repo to be used as a git submodule (https://github.com/sanskrit/sanscript.js/pull/13#issuecomment-668956253), but we could also consider publishing them as an npm module under the `@sanskrit-coders` namespace.

**Comment by vvasuki** [comment]
@psvenk - Which programming language were you particularly interested in adapting sanscirpt to? python and php adaptions exist.

**Comment by psvenk** [comment]
@vvasuki I was initially just playing with implementing it in Haskell but then realized that the specification of schemes would be much too verbose unless I moved it out into separate file in something like JSON or TOML and parsed that using a library such as [aeson](http://hackage.haskell.org/package/aeson-1.5.2.0/docs/Data-Aeson.html) (it seems that there are no libraries that automatically generate Haskell types from the schema, so that would probably entail redefining the schema in terms of Haskell types manually, though this could be eased by using [aeson-schemas](https://hackage.haskell.org/package/aeson-schemas)).

I was able to find the PHP and Java implementations, but https://github.com/sanskrit/sanscript.py is an empty repository. Am I looking in the wrong place?
